src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    this.hidden = false;
```

Comment:
Carnivores shouldn't have a hidden parameter. Only Herbivores should have it. You should replace it to Herbivore class.